src/api/java/com/ldtteam/structurize/api/util/BlockUtils.java

```diff
@@ -376,7 +376,7 @@ public static IBlockState getBlockStateFromStack(final ItemStack stack)
             return Blocks.LAVA.getDefaultState();
         }
 
-        return stack.getItem() instanceof ItemBlock ? ((ItemBlock) stack.getItem()).getBlock().getStateFromMeta(stack.getItemDamage()) : Blocks.GOLD_BLOCK.getDefaultState();
+        return stack.getItem() instanceof ItemBlock ? ((ItemBlock) stack.getItem()).getBlock().getDefaultState() : Blocks.GOLD_BLOCK.getDefaultState();
```

Comment:
@Raycoms just check this

Reply:
No, this doesn't work, we don't want the default state, we want the real state

Reply:
It's deprecated and forwards to this method

Reply:
Hardcoded forward

Reply:
Only in the default block, not in blocks which have metadata

Reply:
Okay will revert it in the second pr